io.ossia.score.yml

```diff
@@ -0,0 +1,333 @@
+id: io.ossia.score
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: ossia-score
+finish-args:
+  # BLE support
+  - --allow=bluetooth
+
+  # Access user's data and media libraries, plug-ins etc
+  - --filesystem=host
+
+  # GPU rendering support
+  - --device=dri
+
+  # Raw input support
+  - --device=input
+
+  # Required for USB & MIDI devices support as well as JACK (/dev/shm) and raw GPIO I/O
+  - --device=all
+
+  # Removable drives support
+  # https://github.com/flatpak/flatpak/issues/2713#issuecomment-1958510347
+  - --filesystem=/media
+  - --filesystem=/run/media
+
+  # X11 is used by default
+  - --socket=x11
+
+  # Wayland still has rendering regressions compared to X11 as of Qt 6.7
+  - --socket=wayland
+
+  # Used by X11
+  - --share=ipc
+
+  # Network access
+  - --share=network
+
+  # Permit PipeWire and user music directory access
+  - --filesystem=xdg-run/pipewire-0
+  - --filesystem=xdg-music
+
+  # PulseAudio is needed for ALSA device access
+  - --socket=pulseaudio
+
+  # Talk to UPower and RTKit services on the session bus
+  - --system-talk-name=org.freedesktop.UPower
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+
+  # Ignore ALSA configuration
+  - --env=ALSA_CONFIG_PATH=
+  - --env=VST_PATH=/app/extensions/Plugins/vst
+  - --env=VST3_PATH=/app/extensions/Plugins/vst3
+
+  # Required to allow screensaver/idle inhibition
+  - --talk-name=org.freedesktop.ScreenSaver
+
+  # For correct cursor scaling under Wayland
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+
+  # Plugins
+  - --env=LV2_PATH=$HOME/.lv2:/app/extensions/Plugins/lv2:/app/lib/lv2
+
+  # Config persistence
+  - --persist=ossia
```

Comment:
it writes `~/ossia` ? This should be fixed upstream. This is ugly.

Reply:
it's also redundant with the broad filesystem permissions.

Reply:
hmm it writes .config/ossia and .cache/ossia, should I put that ?

Reply:
as well as $XDG_DOCUMENTS_DIR/ossia

Reply:
> hmm it writes .config/ossia and .cache/ossia, should I put that ?

It should use XDG_CACHE_HOME and XDG_CONFIG_HOME instead of hardcoding `~/.config` then you don't need to do anything.

Reply:
okay, this is done through Qt's QStandardPaths so I assume the correct folders are already chosen automatically